pimd/pim_bsm.c

```diff
@@ -1362,8 +1352,8 @@ int pim_bsm_process(struct interface *ifp, pim_sgaddr *sg, uint8_t *buf,
 		} else {
 			if (PIM_DEBUG_BSM)
 				zlog_debug(
-					"%s : nofwd_bsm received on %s when accpt_nofwd_bsm false",
-					__func__, bsr_str);
+					"%s : nofwd_bsm received on %pPAs when accpt_nofwd_bsm false",
+					__func__, &bshdr->bsr_addr.addr);
 			pim->bsm_dropped++;
 			pim_ifp->pim_ifstat_ucast_bsm_cfg_miss++;
 			return -1;
```

Comment:
`#if PIM_IPV == 4` 3 lines below (not shown above) should be removed

Reply:
Done